frappe/core/doctype/doctype/doctype.py

```diff
@@ -1740,3 +1740,33 @@ def get_field(doc, fieldname):
 	for field in doc.fields:
 		if field.fieldname == fieldname:
 			return field
+
+
+@frappe.whitelist()
+def set_field_order(doctype, field_order):
+	"""Update field order in doctype"""
+
+	frappe.only_for("System Manager")
```

Comment:
Developer mode should also be checked here? 🤔 

not mentioned in PR but afaik this is intended only for developer mode for now?

Reply:
using `doctype.save()` now, so it will check for developer mode